src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Bulldozer bulldozer = new Bulldozer();
+        Truck truck = new Truck();
+        Excavator excavator = new Excavator();
+        Machine[] machines = new Machine[] {bulldozer, truck, excavator};
+        for (Machine elem : machines) {
```

Comment:
```suggestion
        for (Machine machine : machines) {
```
Don’t use abbreviations for names